app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/twofactorlogin/TwoFactorLoginScreen.kt

```diff
@@ -254,6 +255,9 @@ private fun TwoFactorLoginScreenContent(
                 modifier = Modifier
                     .padding(horizontal = 16.dp)
                     .fillMaxWidth(),
+                keyboardActions = KeyboardActions(
```

Comment:
Super minor but can we lift this up so the modifier is last?

Reply:
done!